build.sbt

```diff
@@ -37,6 +37,7 @@ val baseSettings = Seq(
     "com.chuusai" %% "shapeless" % shapelessVersion,
     "com.twitter" %% "finagle-http" % finagleVersion,
     "org.scala-lang" % "scala-reflect" % scalaVersion.value,
+    "org.scala-lang.modules" %% "scala-xml" % "1.0.5",
```

Comment:
I'd vote for extracting it into a standalone sub-project and using something like [AntiXML](http://anti-xml.org/) instead.